lib/diagnostic.js

```diff
@@ -38,20 +51,30 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {name:"Shadow",
+           species: "cat",
+           breed:"Calico",
+           vocalization: "Meow"
+};
+
 
```

Comment:
I believe js prefers single quotes for object properties, but very close that aside.

```js
let pet = {
  name: 'Beau',
  species: 'Dog',
  breed: 'Lab',
  vocalization: 'bark',
  vocalize: function () {
    return this.vocalization;
  }
};
```